core/templates/dev/head/admin/Admin.js

```diff
@@ -16,64 +16,41 @@
  * @fileoverview Data and controllers for the Oppia admin page.
  */
 
+oppia.constant('ADMIN_TAB_URLS', {
+  ACTIVITIES: '#activities',
+  JOBS: '#jobs',
+  CONFIG: '#config',
+  MISC: '#misc'
+});
+
 oppia.controller('Admin', [
-  '$scope', '$http', 'UrlInterpolationService',
-  function($scope, $http, UrlInterpolationService) {
-    $scope.message = '';
-    $scope.adminHandlerUrl = '/adminhandler';
+  '$scope', '$http', 'UrlInterpolationService', 'ADMIN_TAB_URLS',
+  function($scope, $http, UrlInterpolationService, ADMIN_TAB_URLS) {
     var ADMIN_JOB_OUTPUT_URL_PREFIX = '/adminjoboutput';
-    $scope.adminTopicsCsvDownloadHandlerUrl = '/admintopicscsvdownloadhandler';
+    var adminHandlerUrl = '/adminhandler';
```

Comment:
Worth making these two things constants (e.g. ADMIN_HANDLER_URL), for consistency?